docs/how-to-guides/themes/theme-support.md

```diff
@@ -391,20 +391,38 @@ Some blocks can have padding controls. This is off by default, and requires the
 add_theme_support('custom-spacing');
 ```
 
-## Experimental — Link color control
+## Link color control
+
+Link support has been made stable as part of WordPress 5.8. It's `false` by default and themes can enable it via the [theme.json file](./theme-json.md):
+
+```json
+{
+	"version": 1,
+	"settings": {
+		"color": {
+			"link": true
+		}
+	}
+}
+```
 
-Using the Gutenberg plugin (version 8.3 or later), link color control is available to a number of blocks including Paragraph, Heading, Group, Columns, and Media & Text blocks. This is off by default, and requires the theme to opt in by declaring support:
+Alternatively, opt-in via the old legacy support would also work:
```

Comment:
[This](https://github.com/WordPress/wordpress-develop/blob/28ea7b18cf23e9e373a52cad21a739b030e3fced/src/wp-includes/class-wp-theme-json.php#L1223) has landed in WordPress 5.8. I wonder if we can remove it in WordPress 5.9 or should it live in core forever.

Reply:
We discussed this in the past and the agreement seemed to be that `experimental-link-color` should only be part of the plugin and will be deprecated and removed when the plugin required WordPress 5.8 as the minimum version.

Going to prepare a PR to remove the above from core and make sure it's only part of the plugin.

Reply:
https://github.com/WordPress/wordpress-develop/pull/1458